[Doc]: Update dynamic node registration process

molotkov-and opened the pull request:
### Changelog entry <!-- a user-readable short description of the changes that goes to CHANGELOG.md and Release Notes -->

Update dynamic node registration process. Delete unnecessary settings.

### Changelog category <!-- remove all except one -->

* Documentation (changelog entry is not required)

### Description for reviewers <!-- (optional) description for those who read this PR -->

Update dynamic node registration process. Delete unnecessary settings.

molotkov-and committed:
Fix registration node process for ru doc

molotkov-and committed:
Fix registration node process for en doc

orange13 committed:
add configuration

yurikiselev committed:
Update node_registration_token description

yurikiselev commented on ydb/docs/ru/core/reference/configuration/auth_config.md:
```diff
@@ -376,3 +376,36 @@ auth_config:
+Возможные значения:
+
+- Пустая строка (`""`) — используется режим аутентификации узлов через сертификаты TLS. В этом случае узлы должны использовать сертификаты для аутентификации при регистрации в кластере. Подробнее о настройке аутентификации узлов через сертификаты см. в разделе [Аутентификация и авторизация узлов баз данных](../../devops/configuration-management/configuration-v1/node-authorization.md).
+- Непустая строка — используется режим аутентификации узлов через токен. Узлы баз данных должны предоставить указанный токен при регистрации в кластере.
```
На самом деле тут совсем не любой токен. Раньше можно было указывать только одно значение по сути: `root@builtin` (это же и является дефолтом). Настройку переиспользовали, что бы пустое её значение считалось указанием системе использовать сертификаты для аутентификации дин. нод.

То есть разумных значения тут два: пустое (аутентификация по сертам) и `root@builtin` (это аутентификация из-под системного пользователя как бы).

yurikiselev replied:
С Сашей обсуждали это место, решили описать лаконично. Тут разумное значение токена только `root@builtin`, его использовать сейчас не рекомендуем, а рекомендуем переходить на сертификаты. Это в тексте написали.

azevaykin commented on ydb/docs/ru/core/reference/configuration/auth_config.md:
```diff
@@ -376,3 +376,36 @@ auth_config:
+Возможные значения:
+
+- Пустая строка (`""`) — используется режим аутентификации узлов через сертификаты TLS. В этом случае узлы должны использовать сертификаты для аутентификации при регистрации в кластере. Подробнее о настройке аутентификации узлов через сертификаты см. в разделе [Аутентификация и авторизация узлов баз данных](../../devops/configuration-management/configuration-v1/node-authorization.md).
+- "root@builtin" — используется режим аутентификации узлов через токен субъекта доступа `root@builtin`.
```
нужна ссылка, что такое root@builtin ?

yurikiselev replied:
Он сейчас фигурирует в пяти местах примерно, но нигде не описано, что это. Мы не первые на него ссылаться начали
https://nda.ya.ru/t/xiyW8ZdJ7ZirrL

azevaykin replied:
Давай, хоть минимальное описание добавим про root@builtin

yurikiselev replied:
добавил https://ydb-platform--ydb.viewer.diplodoc.com/ru/reference/configuration/auth_config?revision=pr-29177-69430b7105f8202376ec3d8c349242abe480e88f#node-registration-token

azevaykin replied:
"root@builtin" — режим аутентификации через особый отладочный токен. Данный режим планируется к исключению в будущих релизах и не рекомендуется к использованию.

yurikiselev replied:
Поправил
https://ydb-platform--ydb.viewer.diplodoc.com/ru/reference/configuration/auth_config?revision=pr-29177-356e3ebc61915f69baa0a0da795fcc7e9e942adc#node-registration-token

yurikiselev replied:
и ещё раз https://ydb-platform--ydb.viewer.diplodoc.com/ru/reference/configuration/auth_config?revision=pr-29177-1a2f50ef81e591093d762e4b5c4c8454afae2a48#node-registration-token

yurikiselev committed:
Add info about root@builtin

yurikiselev commented on ydb/docs/ru/core/reference/configuration/auth_config.md:
```diff
@@ -376,3 +376,36 @@ auth_config:
     ||
 |#
+
+## Конфигурация токена регистрации узлов {#node-registration-token}
+
+{{ ydb-short-name }} позволяет настраивать тип аутентификации узлов баз данных при их регистрации в кластере. Настройка это типа осуществляется через параметр `node_registration_token` секции `auth_config`.
+
+#|
+|| Параметр | Описание ||
+|| node_registration_token
```
сделаем отдельно

yurikiselev committed:
typo

yurikiselev committed:
Clarify node_registration_token configuration instruction

yurikiselev committed:
Update auth_config.md

yurikiselev committed:
Review fix – rewrite root@builtin descr.

yurikiselev committed:
Update auth_config.md

yurikiselev committed:
Update auth_config.md

yurikiselev committed:
Update auth_config.md

yurikiselev committed:
Update auth_config.md

Clarified the default value for error refresh time and added context for node registration token configuration.

yurikiselev committed:
Fix auth_config.md

yurikiselev commented:
Перевёл правки на страницу https://ydb-platform--ydb.viewer.diplodoc.com/en/reference/configuration/auth_config?revision=pr-29177-704c00294cde3d69737ff2651b3904404acdc600#node-registration-token
